### PR TITLE
feat(performance): various performance, SEO, and analytics tweaks

### PR DESCRIPTION
**Pull request by admoya**

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <link rel="icon" href="%sveltekit.assets%/favicon.png" />
     <link
       rel="stylesheet"
-      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0..1,0" />
```
Comment (admoya) [inline src/app.html:9]: We are only using this size icon, and this significantly reduces the size of the font

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -5,10 +5,11 @@
   import { browser } from "$app/environment";
 
   // Google Analytics
-  function gtag() {
-    window.dataLayer.push(arguments);
-  }
   if (browser) {
+    // @ts-ignore
+    window.gtag = function() {
```
Comment (admoya) [inline src/routes/+layout.svelte:10]: to make `gtag` available globally

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
 <svelte:head>
   <title>KHA Home</title>
+  <meta name="description" content="Welcome to Kendale Homeowners Association, a community in the Village of Kendale, Kendale South, and Kendale Country Club Estates. Learn about our services including community maintenance, local events, and community information.">
```
Comment (admoya) [inline src/routes/+page.svelte:11]: Meta tags for SEO

File: src/lib/assets/park.webp
Comment (admoya) [inline src/lib/assets/park.webp:1]: Compressed the park image for faster loads

**Review by exscruzme — APPROVED** [review]

**Pull request merged**